Experimental/DistributedAuthoritySample/Assets/Scenes/TownMarket_Test_ScenarioWeek.unity.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Since we're using this scene, let's rename it and its associated subscene folder.

To comment on the scene itself, I'm seeing a potential optimization of excluding colliders on prefabs that are not inside the main market area. For example, there's colliders on the in-the-distance houses + cave area. Hopefully its an easy swap of prefabs in those cases.

Reply:
I will rename the scene. Regarding the colliders I can deactivate them or remove them manually, but not really swap the prefab? I am unsure what you mean.

Reply:
I wouldn't store that as a scene change. I would instead have a collider-less (just geometry) & a variant with colliders.

If you have the variant, you can right click into a prefab instance in the scene, navigate to Prefab/ and use Replace... and you could pick the variant -- it preserves the transform information.